### PR TITLE
add vedio-page

### PR DESCRIPTION
**Pull request by archit7448**
### What I have done 

- Add video-Page
- Add logIn and SignUp Page 
- Work on responsivness

### What to check 

- VideoPlayer.jsx 
- Login.jsx
- signUp.jsx
- auth.js

### CODE REVIEW

File: src/pages/Login/login.jsx [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+import "./login.css";
 export const LogInPage = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { LoginHandler } = useAuth();
+  const guestHandler = () => {
+    setEmail("architsingh@gmail.com");
+    setPassword("architSingh123");
+  };
```
Comment (bhushannaikaroskar) [inline src/pages/Login/login.jsx]: You can name this as setGuestCredentials or something similar. This way it's understandable that what he function does. This is a personal suggestion you can keep this as well no issues.

**Review by bhushannaikaroskar — APPROVED** [review]
Hey Archit reviewed your code. Didn't find any major changes. One thing I would say is do check your css before merging because at some place you have use 0px, 0rem and at some place used 0 so you can make every value as 0 for consistency. Rest I think your code is good. Approving your merge

File: src/pages/videoPlayer/videoPlayer.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { useParams } from "react-router-dom";
+import { useData } from "../../context/data";
+import { Header } from "../../Components/header/header";
+import { Sidebar } from "../../Components/sidebar/sidebar";
```
Comment (amartya-rs) [inline src/pages/videoPlayer/videoPlayer.jsx]: suggestion: you can export components from a common file say index.js

**Review by amartya-rs — APPROVED** [review]
Hey Archit, your code looks fine to me. You can merge it with the suggested changes.

**Pull request merged**